src/assets/scripts/client/parsers/aircraftCommandParser/aircraftCommandMap.js

```diff
@@ -123,3 +213,14 @@ export const AIRCRAFT_COMMAND_MAP = {
  * @final
  */
 export const EXPEDITE = ['expedite', 'x'];
+
+/**
+ * Get the name of a command when given any of that command's aliases
+ *
+ * @function getCommandNameFromAlias
+ * @param alias {string}
+ * @return {string}
+ */
+export function getCommandNameFromAlias(alias) {
```

Comment:
`.findCommandNameWithAlias()`

and could we change the param name to: `commandAlias`

Reply:
works for me. It's a helper function though-- you don't mean to add it to a class or anything do you? Just since you used a dot in front.

Reply:
Oh, no, it's just fine as a function.

Reply:
done